dotCMS/src/main/java/com/dotcms/vanityurl/model/DefaultVanityUrl.java

```diff
@@ -1,6 +1,7 @@
 package com.dotcms.vanityurl.model;
 
 import com.dotcms.contenttype.model.type.VanityUrlContentType;
+import com.dotmarketing.exception.DotRuntimeException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.exception.DotRuntimeException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)